lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -47,8 +47,15 @@
         description:
         - Type of network device network manages.
         - Required when creating a network.
-        choices: [appliance, combined, switch, wireless]
+        - As of Ansible 2.8, combined type has moved to the types parameter.
+        choices: [appliance, switch, wireless]
```

Comment:
We tend to leave spaces around list content in YAML ofr readability, but it's debatable as PEP8 does not allow this in Python.
```suggestion
        choices: [ appliance, switch, wireless ]
```